.circleci/config.yml

```diff
@@ -207,7 +207,7 @@ jobs:
             command: |
               set -x
               make get_tools
-              make get_vendor_deps
+              make get_vendor_deps_ci
```

Comment:
I remember we had something like that in Tendermint, but switched to `make get_vendor_deps`. Does this mean we should do the same change in Tendermint?

Reply:
Probably. We made that change before we ensured the lock file is correct, but not that we do, we can go back to how we used to do it Tendermint!